### ISSUE TITLE
Crafting paper with the config for easier recipe give back 2 stone slabs, even if the recipe doesn't use them

### ISSUE THREAD

**Issue by Alessandro404** [Blood-Asp/GT5-Unofficial#900]
using version 5.10.40

**Comment by Blood-Asp** [comment]
Did you change the recipe with Minetweaker? Or is it different in 1.10.2 by default? Or does some other mod add alternative paper recipes?

**Comment by Alessandro404** [comment]
No I didn't changed the recipe, I only set the config inside gregtech, and no I had to change it in 10.2, but from memory is the same than in 1.7.10

**Comment by MauveCloud** [comment]
I can reproduce this in 1.10.2, and also get extra slabs when uncrating paper with a crowbar (regardless of whether the config is set to easy or hard recipe for the paper).

However, looking at https://github.com/Blood-Asp/GT5-Unofficial/blob/unstable/src/main/java/gregtech/loaders/misc/GT_Achievements.java#L345 I'm confused that this doesn't happen in 5.09.  Testing with the hard recipes reveals that in 5.09.29pre, slabs are not returned.  

Also, I'm not certain alternative paper recipes from other mods would work - in my 5.09.29pre world, I've got an option for "bookToPaperRecipe" in endercore.cfg set to true, but no such recipe shows up.

Edit: I was going to file a bug about the slabs not being returned in 5.09.29pre, but apparently one of my other mods is interfering with that - when running with a bare minimum of mods (GT, IC2, CCC, and NEI), it does return the slabs, but not only when the hard recipe is enabled - the easy recipe and opening crates of paper with a crowbar also add stone slabs to my inventory.

I can think of a few options for fixing this:
1. Adjust the recipe to reuse the slabs (so they remain in the crafting grid instead of being re-added to inventory afterward).  Minetweaker offers a way to do this, but idk if the GregTech API does.
2. Verify that the paper was crafted using slabs before adding them to the player's inventory - I confirmed that the event includes details of the crafting recipe used:
1.7.10: https://github.com/MinecraftForge/MinecraftForge/blob/1.7.10/fml/src/main/java/cpw/mods/fml/common/gameevent/PlayerEvent.java#L28
1.10.2: https://github.com/MinecraftForge/MinecraftForge/blob/1.10.x/src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java#L47
3. Skip returning the slabs - this may reduce realism, but stone is a renewable resource in Minecraft; there are numerous designs for cobblestone/smooth stone generators that can be built in vanilla Minecraft, and the GT Rock Breaker is one of two GT machines that can be crafted with no circuits in the recipe.
4. Add a new metatool, maybe called "paper press", made from two stone slabs in the same positions as the hard recipe (and maybe allow a few types of metal plates as well), and let the existing GT metatool durability system take care of making it reusable for a while (I figure it should eventually wear out and/or build up too much paper dust to be usable)
